### ISSUE TITLE
imported server cert not work

### ISSUE THREAD

**Issue by kaala** [picoHz/taxy#70]
I imported my self signed certificate, and it did not work.

the certificate is generated using these commands.
openssl genrsa -out ssl.key 2048
openssl req -new -key ssl.key -out ssl.csr
openssl x509 -req -days 1460 -in ssl.csr -signkey ssl.key -out ssl.crt

debug log show this error, what does this mean?
![image](https://github.com/picoHz/taxy/assets/246542/eabca8d5-53b0-4ceb-b264-a6226542a0fb)

**Comment by picoHz** [comment]
@kaala 
Currently, taxy only accepts a specific kind of private key, and subjectAltName must also be set. 
I plan to address these limitations.

Here is the way to generate a certificate that works in the current version:

```bash
openssl genpkey -algorithm EC \
  -pkeyopt ec_paramgen_curve:P-256 \
  -pkeyopt ec_param_enc:named_curve | \
openssl pkcs8 -topk8 -nocrypt -outform pem > ssl.key

openssl req -new -key ssl.key -out ssl.csr

echo "subjectAltName = DNS:yourdomain.com" > san.cnf
openssl x509 -req -days 1460 -in ssl.csr -signkey ssl.key -out ssl.crt -extfile san.cnf
```

**Comment by picoHz** [comment]
Fixed in [v0.3.29](https://github.com/picoHz/taxy/releases/tag/v0.3.29)
